src/TraceEvent/Parsers/ClrTraceEventParser.cs

```diff
@@ -2162,13 +2174,17 @@ static private ThreadPoolMinMaxThreadsTraceData ThreadPoolMinMaxThreadsTemplate(
         {                  // action, eventid, taskid, taskName, taskGuid, opcode, opcodeName, providerGuid, providerName
             return new ThreadPoolMinMaxThreadsTraceData(action, 59, 38, "ThreadPoolMinMaxThreads", ThreadPoolMinMaxThreadsTaskGuid, 0, "Info", ProviderGuid, ProviderName);
         }
+        static private ContentionLockCreatedTraceData ContentionLockCreatedTemplate(Action<ContentionLockCreatedTraceData> action)
+        {                  // action, eventid, taskid, taskName, taskGuid, opcode, opcodeName, providerGuid, providerName
+            return new ContentionLockCreatedTraceData(action, 90, 8, "Contention", ContentionTaskGuid, 11, "LockCreated", ProviderGuid, ProviderName);
```

Comment:
It looks like this will show up as a `Contention/Info` event and not a `LockCreated` event.  To fix this, you will want to create `LockCreated` opcode, and use that.  Unless I've missed something, this is the event definition, using the `win:Info` opcode:

```
                    <event value="90" version="0" level="win:Informational"  template="LockCreated"
                           keywords ="ContentionKeyword"  opcode="win:Info"
                           task="Contention"
                           symbol="LockCreated" message="$(string.RuntimePublisher.LockCreatedEventMessage)"/>
```